packages/hardhat-plugin/test/verify.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable import/no-unused-modules */
+import type { Etherscan } from "@nomicfoundation/hardhat-verify/etherscan";
+
+import { assert } from "chai";
+
+import { getApiKeyAndUrls } from "../src/utils/getApiKeyAndUrls";
+import { verifyEtherscanContract } from "../src/utils/verifyEtherscanContract";
+
+import { useEphemeralIgnitionProject } from "./use-ignition-project";
+
+describe("verify", function () {
```

Comment:
Maybe create a verify folder and split the apikeylist tests from the fuller tests?